todo-app/src/toDos-DB.js

```diff
@@ -0,0 +1,18 @@
+export let ToDos=[
+    {
+        todoDesc: "Get Out of Bed",
+        toDoStatus: "Done"
```

Comment:
You could have used a `boolean` property here for `toDoStatus`, could be either `true` or `false`.